recipes/eigency/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "eigency" %}
+{% set version = "2.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/wouterboomsma/eigency/archive/refs/tags/{{ version }}.tar.gz
+  sha256: fff953cc40e2ff658eebdaa4c337ebe869984b0556656126a1e1d2dd399c05e7
+  patches:
+  - 0001-User-can-use-system-eigen.patch
+
+build:
+  script: EIGEN_INC=${PREFIX}/include/eigen3 {{ PYTHON }} -m pip install --no-deps . -vv  # [not win]
+  script: set EIGEN_INC=%PREFIX%\Library\include\eigen3& {{ PYTHON }} -m pip install --no-deps . -vv  # [win]
+  number: 0
```

Let's use `script_env` instead of jumping through commandline hoops:
```suggestion
build:
  script_env:
    - EIGEN_INC=${PREFIX}/include/eigen3
  script: {{ PYTHON }} -m pip install --no-deps . -vv
  number: 0
```